Vostok.Hosting.Abstractions/IVostokHostExtensions.cs

```diff
@@ -36,5 +36,10 @@ public interface IVostokHostExtensions
         /// Returns all types registered without key.
         /// </summary>
         IEnumerable<(Type, object)> GetAll();
+        
+        /// <summary>
+        /// Returns all types registered.
+        /// </summary>
+        IEnumerable<(Type Type, object Extension, string Key)> GetAll(bool withoutKeys, bool withKeys);
```

Comment:
Parameters are also unclear, we already have `GetAll()` which is equivalent to `GetAll(true, false)`

So we need only simple `GetAllKeyed()` method, which is already implemented [here](https://github.com/vostok/hosting/blob/6f4242854ae4ff7ba13dbedd150bd7341aa9e67e/Vostok.Hosting/Components/HostExtensions/HostExtensions.cs#L55).